### ISSUE TITLE
Please include requirements.txt

### ISSUE THREAD

**Issue by robmarkcole** [jrosebr1/simple-keras-rest-api#11]
As title. I think https://github.com/jrosebr1/simple-keras-rest-api/issues/7 may be due to changes in newer version of tensorflow

**Comment by btphan95** [comment]
Yes, it seems like the newer version of TensorFlow is not compatible with the Keras methods used in this repository. 
Here are the versions of Keras and TF that worked for me:
Keras 2.2.4
TF 1.13.1
